source/ci_source/providers/Fake.ts

```diff
@@ -6,8 +6,8 @@ export class FakeCI implements CISource {
 
   constructor(env: Env) {
     const defaults = {
-      repo: env.DANGER_TEST_REPO || "artsy/emission",
-      pr: env.DANGER_TEST_PR || "327"
+      repo: env.DANGER_TEST_REPO || "artsy/emission", // TODO: default to empty string ?
+      pr: env.DANGER_TEST_PR || "327" // TODO: default to empty string ?
```

Comment:
Hrm, I generally like to have test data that is real for my desalts during dev - I'm still OK with keeping these as this repo mainly because it keeps it consistent with all the fixture data that is inside the tests